Hush/Assets/Scripts/Game/GameManager.cs

```diff
@@ -49,5 +49,12 @@ public void SetGameState(GameState gameState)
             ApplyEcholocationState();
             ApplySpellState();
         }
+
+        public void NewGame()
```

Comment:
NewGame() should be in SceneManager and should call something like GameManager.Instance.ResetGameState()

which does what you do at line 55